chapter_01.md

```diff
@@ -1,2 +1,550 @@
+
+In the case of our scenario, dealing with Call for Proposals is critical to bootstrap a conference. 
+There is substantial work around accepting proposals, reviewing them and making sure that the conference has an interesting agenda. 
+
+If we are confident that implementing the Call For Proposal functionality first will give us immediate wins, we need to estimate how big and how complex the effort is. 
+
+DDD proposes the concept of Bounded Context as a well-scoped set of functionalities that belong together. These functionalities usually map how a Domain Expert (Subject Matter Expert) will do the work if there is no software available. In order to plan, design, and implement these functionalities, a team is assembled with Domain Experts that will work hand in hand with Software Engineers. Most of the time, a Domain Expert will know what Bounded Contexts already exist and what are they responsible for. From a DDD perspective, it is extremely important to not make up these Bounded Contexts a technical boundary that is imposed on Domain Experts. 
+
+A Bounded Context will expose a well-defined set of APIs so other Teams from different Bounded Contexts can consume and interact with the Call For Proposals functionality. 
+
+The Call for Proposals Bounded Context will enable a team to implement all the needed functionality independently of other teams. This team will be in charge of the entire lifecycle of the project, from designing it, implementing it, running it for the rest of the company and for your customers to consume it. 
+
+As soon as you start designing the Call for Proposal functionality you realize that you will need to consume and interact with other teams. Very early on the following Bounded Contexts are identified:
+
+![chapter_01_03](images/chapter_01_03.png)
+
+Each of these Bounded Contexts will be owned by different teams and we need to make sure that they have enough autonomy to make progress, create new versions with new features, and deploy concrete software components to our customer’s environments. 
+
+On a practical side, each Bounded Context will be implemented as one or a set of services that implement the context features. There is no need to be strict about the number of services that will compose a Bounded Context, but usually, there is one that is in charge of exposing a set of APIs to the outside world. 
+
+For the purpose of this example, a single service will implement all the Call For Proposals Bounded Context logic and the team behind this service will be responsible for designing its APIs, choosing the frameworks that they are going to use, and actually deploying it into a live environment. 
+
+Going deep into practical details, there are a couple of best practices shared by many companies and tools: 
+- One Repository /  One Service + Continuous Delivery
+- Open APIs
+
+
+#### One Repository / One Service + Continuous Delivery
```

Comment:
Based on my understanding of the text, these are conventions that Jenkins X uses and that you will mention later on. Should we maybe move part of this section to the cloud chapter where we have a CI/CD section? We could simplify the content in here by mentioning that details about these conventions, and about jenkinsx are given on the cloud chapter. 

I'm afraid we might lose focus on DDD if we started detailing these important ci/cd practices here.

Reply:
Ok, I might have not been clear enough in the text, most of those recommendations/best practices come from the Accelerate book, which was created based on a crazy amount of research and industry surveys. 
If you take as this chapter is about DDD, then the focus of these topics is wrong. My original intention was more like. If you know DDD, this is what comes next, if you try to implement it. If you don't have CI/CD, a Gateway, SSO in mind for implementing DDD then you will run in a lot of troubles. So moving towards other topics that come as result of using DDD for a microservices architecture should be ok for this chapter. Let me know what you think @kmacedovarela . I can make it clear in the intro of the chapter.